Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -432,6 +433,7 @@ extension TelemetryWrapper {
         case readingPanel = "reading-panel"
         case downloadsPanel = "downloads-panel"
         case syncPanel = "sync-panel"
+        case yourLibrarySection = "your-library-section"
```

Comment:
How are these raw values determined?

Reply:
That's a great question. As far as I've been told, we're free to make them up, basically, given that, in our current `metrics.yaml` we're explicit about what these values are. In my telemetry overhaul, I have a proposal that will make all the categorizations and such consistent such that, when adding values, they are very logical.